openslides_backend/migrations/migrations/0042_remove_template_fields.py

```diff
@@ -0,0 +1,384 @@
+                    elif strategy is FieldStrategy.MergeToJSON:
+                        # merge values together into a single list of key-value pairs
+                        new_value.append((replacement, structured_value))
+                    elif strategy in (
+                        FieldStrategy.MoveToMeetingUser,
+                        FieldStrategy.MoveToMeetingUserAndReplace,
+                    ):
+                        # move value to new field in meeting_user
+                        meeting_id = int(replacement)
+                        key = MeetingUserKey(meeting_id, model["id"])
+                        # replace user ids with meeting_user ids, if necessary
+                        self.meeting_users[key][new_field] = (
+                            structured_value
+                            if strategy is FieldStrategy.MoveToMeetingUser
+                            else get_meeting_user_ids(meeting_id, structured_value)
+                        )
+                    else:
+                        raise ValueError()
+
+            if new_value:
+                if strategy is FieldStrategy.MergeToJSON:
+                    # make dict from key-value pairs
+                    update[new_field] = dict(new_value)
+                else:
+                    update[new_field] = new_value
+        return update
+
+    def resolve_strategy(  # noqa: E731
+        self, strategy: FieldStrategy | ParametrizedFieldStrategy
+    ) -> Tuple[FieldStrategy, FieldNameFunc]:
+        """
+        Resolves a (parametrized) strategy to a tuple of strategy and the new field name.
+        """
+        if isinstance(strategy, dict):
+            resolved_strategy = strategy["strategy"]
+            if isinstance(strategy["name"], str):
+                name_func = lambda _: strategy["name"]  # noqa: E731
+            elif isinstance(strategy["name"], dict):
+                name_map = strategy["name"]
+                name_func = (  # noqa: E731
+                    lambda field: name_map[field]
+                    if field in name_map
+                    else field.replace("$", "")
+                )
+            else:
+                name_func = strategy["name"]
+        else:
+            resolved_strategy = strategy
+            if resolved_strategy is FieldStrategy.Rename:
+                name_func = lambda field: field.replace("$", "")  # noqa: E731
+            elif resolved_strategy in (
+                FieldStrategy.Merge,
+                FieldStrategy.MergeToJSON,
+                FieldStrategy.MoveToMeetingUser,
+                FieldStrategy.MoveToMeetingUserAndReplace,
+            ):
+                name_func = lambda field: field.replace("_$", "")  # noqa: E731
+            elif resolved_strategy is FieldStrategy.ReplaceWithMeetingUsers:
+                name_func = lambda field: f"meeting_{field}"  # noqa: E731
+            else:
+                name_func = lambda field: field  # noqa: E731
```

Comment:
Also here, raise an exception. This only occurs by a template field without a strategy